templates/js/controllers/userController.mongo.js

```diff
@@ -1,7 +1,7 @@
-import {User} from '../models';
+import { User } from '../models';
 
-export const getAllUsers = (req, res, next) => {
+export const getAllUsers = (_, res) => {
```

Comment:
Same thing here with `next` middleware. Also, associating `_` with request might confuse devs with `lodash`

Reply:
For this one i plan on adding error handling for the `.find()` method and then i will be able to use `next(err)`
As for `res` i cant remember if the linter was upset about that one or i just changed it for convention, i can revert this back to `res` if linter is ok